config/locales/en/documents/show.yml

```diff
@@ -69,3 +69,10 @@ en:
           chosen: "Image “%{file}” has been selected as the lead image"
           removed: "Image “%{file}” has been deselected. Document will now use the default lead image"
           deleted: "Image “%{file}” has been deleted. Document wil now use the default lead image"
+        topics_updated: Topics updated on all editions
+        topic_update_error:
+          title: Something has gone wrong
+          description: Something went wrong when saving your changes. Please try again or [raise a support request](https://support.publishing.service.gov.uk/technical_fault_report/new).
```

Comment:
Do we repeat this translation in a few places?

Reply:
We have a lot of translations that are similar, but this one is still unique. I think we should keep them separate to allow a content designer to evolve them separately.